api4/system.go

```diff
@@ -402,6 +403,27 @@ func getAnalytics(c *Context, w http.ResponseWriter, r *http.Request) {
 	}
 }
 
+func getLatestVersion(c *Context, w http.ResponseWriter, r *http.Request) {
+	if *c.App.Config().ExperimentalSettings.RestrictSystemAdmin {
+		c.Err = model.NewAppError("latestVersion", "api.restricted_system_admin", nil, "", http.StatusForbidden)
+		return
+	}
```

Comment:
What was the requirement around this `RestrictSystemAdmin` again? I need my memory jogged here.

Reply:
https://docs.mattermost.com/configure/configuration-settings.html?highlight=restrictsystemadmin#restrict-system-admin 
```
Restricts the System Admin from viewing and modifying a subset of server configuration settings from the System Console. Not recommended for use in on-prem installations. This is intended to support Mattermost Private Cloud in giving the System Admin role to users but restricting certain actions only for Cloud Admins.
```